tests/integration/ethernet_mtu_test.py

```diff
@@ -30,6 +30,7 @@
 from libnmstate.error import NmstateVerificationError
 
 from .testlib import assertlib
+from .testlib.iproutelib import ip_monitor_assert_stable_link_up
```

Comment:
Please sort the imports. I think you need to swap with line 34.

Reply:
Done! Thanks